Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN adduser --disabled-password --gecos '' pathfinder 
 
 # INSTALL PACKAGES
-RUN apt-get update && \
+RUN apt-get update --fix-missing&& \
```

Comment:
Why did you take this out? Needed to support container on Mac/etc

Reply:
that should not have been removed....

Reply:
it was not really removed, just changed to add the --fix-missing so I could update packages properly